sklearn/linear_model/sgd_fast.pyx

```diff
@@ -319,7 +319,7 @@ cdef class SquaredEpsilonInsensitive(Regression):
         z = y - p
         if z > self.epsilon:
             return -2 * (z - self.epsilon)
-        elif z < self.epsilon:
+        elif z < -self.epsilon:
```

Comment:
oh boy on did this happen. Any chance you can catch this with a test?

this should be documented in the what's new bug fix section.

thx